plugins/pubsub/setup.py

```diff
@@ -15,7 +15,8 @@
     py_modules=['alerta_pubsub'],
     install_requires=[
         'google-cloud-pubsub',
-        'oauth2client'
+        'oauth2client',
+        'grpcio==1.30.0'
```

Comment:
It is bad practice to pin packages to a version in `setup.py`. Package versions should be pinned in `requirements.txt` in future.